packages/contracts/contracts/GenArt721CoreV3.sol

```diff
@@ -1479,6 +1527,18 @@ contract GenArt721CoreV3 is
         scriptCount = project.scriptCount;
     }
 
+    /**
+     * @notice Returns the compressed form of a string in bytes using solady LibZip's flz compress algorithm.
+     * @param _script Script to be compressed. Required to be a non-empty string, but no further validaton is performed.
+     * @return bytes compressed bytes
+     */
+    function getProjectScriptCompressed(
```

Comment:
wdyt about naming this `getCompressed` or `compressScript` or something? I could imagine someone thinking this name is indicating "get the project script for a script that was uploaded as compressed"?

Reply:
Good call! Updated this to `getCompressed`